R/plot_taxa.R

```diff
@@ -1236,51 +1236,72 @@ plot_taxa_abund <- function(data,
 
 
 
-#' Plot sites on US map
+#' Work-in-progress upgraded plot_sites
```

Comment:
@sokole @RafaelRangel0 is this title change intended? It looks like it may have been a placeholder while development was in progress.

Reply:
this is not in the pull request. Looks like you were looking at an older commit that I reverted.